src/ctapipe/conftest.py

```diff
@@ -695,15 +711,50 @@ def dl1_mon_pointing_file(dl1_file, dl1_tmp_path):
     table = Table({"time": time_mon, "azimuth": az, "altitude": alt})
 
     for tel_id in subarray.tel:
-        write_table(table, path, f"/dl0/monitoring/telescope/pointing/tel_{tel_id:03d}")
+        write_table(
+            table,
+            path,
+            f"/dl1/monitoring/telescope/calibration/pointing/tel_{tel_id:03d}",
```

Comment:
You changed the path here to `dl1`

Reply:
The story is a bit more complicated.

In older versions of ctapipe, we already filled simulation pointing into `dl1/pointing/...`, see the backwards compatibility layer here:

https://github.com/cta-observatory/ctapipe/blob/cc9c770554e76053b47140a5023ac7651eb33726/src/ctapipe/io/hdf5eventsource.py#L237-L258

Currently, the `PointingInterpolator` expects the pointing in `dl0/...`:
https://github.com/cta-observatory/ctapipe/blob/cc9c770554e76053b47140a5023ac7651eb33726/src/ctapipe/monitoring/interpolation.py#L154

So it's not as easy as just putting the new pointing now into `dl1/`.

Either we need to drop support for the old data or improve compatibility layer.

Reply:
Reverted in the new version, please resolve the conversation if sufficient.